recordset.go

```diff
@@ -102,6 +103,24 @@ func (rcs *Recordset) IsActive() bool {
 	return rcs.active.Get()
 }
 
+var ErrRecordsetClosed = errors.New("aerospike: Recordset closed")
+
+// Read reads the next record from the Recordset. If the Recordset has been
+// closed, it returns ErrRecordsetClosed.
+func (rcs *Recordset) Read() (record *Record, err error) {
+	var ok bool
+	select {
+	case <-rcs.cancelled:
+	case record, ok = <-rcs.Records:
+	case err, ok = <-rcs.Errors:
```

Comment:
And that's fine. They called Close – they already don't care what they're discarding in your buffered channel. Why does it matter if we throw records away after we've closed the record set?